### DISCUSSION TITLE
Installing Security Onion on Lenovo x3650 M5, error can't kickstart from /dev/sdc1:/ks.cfg

### DISCUSSION THREAD

**Discussion by rserio** [Security-Onion-Solutions/securityonion#12370] [2.4] [answered]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

other (please provide detail below)

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

2

### RAM

768

### Storage for /

110GB

### Storage for /nsm

7.4 TB

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

This is a brand new install on this server, it will ultimately be a distributed node, as this is going to be my span/traffic node.  I have installed the first machine in my vCenter and it is running fine, when I try to install on this machine, I keep seeing the following errors, which I have added to the file list.
![Screenshot from 2024-02-16 10-49-50](https://github.com/Security-Onion-Solutions/securityonion/assets/73540677/4032f1d6-dcfe-43d4-97b9-e7a20523a02c)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by reyesj2** [comment]
Did you verify the iso when you downloaded it? 
Are you virtually mounting the ISO or do you have a physical USB plugged into the server to boot from?

I think I recall having run into the issue in your screenshot before, but that was when trying to point the server to a custom kickstart file that I placed on a USB. Are you attempting to use a custom kickstart ?

**Comment by rserio** [comment]
I have verified the ISO, and I have created a USB boot disk, with balena and rufus, the one made by rufus I can actually see the text menu for installation.

**Comment by rserio** [comment, marked as answer]
OK so I think it is related to the local RAID controller, so I have been able to install Rocky and then use the github which has other issues, which I will open another issue.
